docs/docs/configuration/providers/index.md

```diff
@@ -10,14 +10,14 @@ Valid providers are :
 
 - [Google](google.md) _default_
 - [Azure](azure.md)
+- [Microsoft Entra ID](ms_entra_id.md)
```

Comment:
Please keep this list (other than Google, which is the default) sorted alphabetically